aws/gandalf.tf

```diff
@@ -3,10 +3,55 @@ resource "aws_instance" "gandalf" {
   instance_type = "t2.medium"
   subnet_id = "${aws_subnet.public.0.id}"
   associate_public_ip_address = true
-  security_groups = ["${aws_security_group.default.id}", "${aws_security_group.gandalf.id}"]
+  security_groups = [
+    "${aws_security_group.default.id}",
+    "${aws_security_group.gandalf.id}"
+  ]
   key_name = "${var.key_pair_name}"
   tags = {
     Name = "${var.env}-tsuru-gandalf"
   }
 }
 
+resource "aws_security_group" "gandalf" {
+  name = "${var.env}-tsuru-gandalf"
+  description = "Security group for Gandalf instance that allows SSH access from internet"
```

Comment:
This description is outdated. The security group allows outbound traffic, SSH access from anyware, 8080 from api hosts, and 3232 (archive) from docker and api.